src/pages/changelog.jsx

```
@@ -275,66 +287,138 @@ export default function Changelog() {
-                            p: ({node, ...props}) => <p className="my-3 text-gray-200 leading-relaxed" {...props} />,
-                            ul: ({node, ...props}) => <ul className="list-disc pl-6 my-3 space-y-2" {...props} />,
-                            li: ({node, ...props}) => <li className="text-gray-300 pb-1" {...props} />,
-                            a: ({node, ...props}) => <a className="text-blue-400 hover:text-blue-300 underline" {...props} />,
-                            code: ({node, ...props}) => <code className="bg-gray-800 px-1 py-0.5 rounded text-sm text-cyan-300" {...props} />,
-                            blockquote: ({node, ...props}) => <blockquote className="border-l-4 border-gray-600 pl-4 italic text-gray-400 my-3" {...props} />,
-                            strong: ({node, ...props}) => <strong className="font-bold text-white" {...props} />,
+                            h1: ({ node, ...props }) => (
+                              <h1
+                                className="text-3xl font-bold my-6 pb-3 border-b border-gray-700 text-blue-400 leading-relaxed"
+                                {...props}
+                              />
```

ESLint

[error] Headings must have content and the content must be accessible by a screen reader.

(jsx-a11y/heading-has-content)